modules/core/src/v2/baseCoin.ts

```diff
@@ -420,5 +420,18 @@ export abstract class BaseCoin {
     return m === 2 && n === 3;
   }
 
+  /**
+   * check if a string can safely be converted to a BigNumber
+   * @param str the string in question
+   * @returns the string if it is safe otherwise an empty string
+   */
+  safeStringNumber(str: string): string {
```

Comment:
Is this used anywhere? Why is this better than try/catch? And why is it on baseCoin? 🤔

Reply:
No this isn't used anywhere. I stopped using this approach when I thought about some of the things you mentioned. I thought I had removed it, but guess I forgot. Thanks for catching this.